docs/development-tips.md

```diff
@@ -54,3 +54,13 @@ The structure looks like this:
 !!! note
     After running `gulp build` the site's assets are copied over to `cfgov\static_built`,
     ready to be served by Django.
+
+### TIP: Installing new frontend dependencies
+- Use `yarn add new_dep@se.m.ver` to install new dependencies.
+- If you can't do this for some reason or are looking to freshen all dependencies,
+  you will need to edit `.yarnrc`, commenting out the two included flags before
+  proceeding with your installation or update.
+- In the rare but observed case that `yarn add new_dep@se.m.ver` doesn't add
+  every needed package to the offline cache, you likely need to first run
```

Comment:
And someone opening a PR would know this was the case because github actions would fail on installing dependencies? In https://github.com/cfpb/cfgov-refresh/pull/5617 I ran the updates and built the project and all seemed good till it go to the PR and you fixed it. Is there something I would see locally before running checks on the PR that indicates everything didn't install?

Reply:
Following the `yarn add` workflow, that situation shouldn't be encountered, but you could certainly test locally by running `rm -rf node_modules && yarn cache clean && yarn`

Reply:
heh pushed a solution to this in b118cf2